.travis.yml

```diff
@@ -113,13 +113,15 @@ matrix:
         - source_pkg gcc
         - arm-none-eabi-gcc --version
         - pip install --user --upgrade cmake
+        - pip install --user --upgrade mbed-tools
         - >-
          git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os-example-blinky.git;
          cd mbed-os-example-blinky;
```

Comment:
We'll need to add .mbed file

Reply:
I'd suggest we fix it here for now until we can all discuss what we want to do in the tools going forward (perhaps on Wednesday when @Patater returns from holiday).